prog/minio/minio_cluster_nexus.rb

```diff
@@ -44,22 +53,6 @@ def before_run
     end
   end
 
-  label def start
-    register_deadline(:wait, 10 * 60)
```

Comment:
It seems we don't register deadline anymore in here or in pool nexus. Only server nexus creates deadlines, which covers most places but not all (e.g. configure_dns_records). Is this intentional? If we want to reduce deadline registrations, maybe we can remove the server one and keep the cluster one as that would cover the most places.

Reply:
This was missed when removing the state. Thanks!

Reply:
I saw you added the one in the pool back. How about the one in the cluster?

Reply:
I have added the one in the cluster back https://github.com/ubicloud/ubicloud/pull/1044/files#diff-6ed3d0438d850729925ee9a9d9ec6ad8e11f4e8c3a37ebe6ce278d5d7981968eR57

Reply:
sorry, misread the change.